docs/whatsnew/2.0.rst

```diff
@@ -249,26 +236,87 @@ to atomic mass unit (amu). For example::
-    >>> g = Gaussian1D(amplitude=1*u.J, mean=1*u.m, stddev=0.1*u.m)
-    >>> g([3, 4, 5.5] * u.cm)
-    <Quantity [  3.70353198e-21,  9.72098502e-21,  4.05703276e-20] J>
+.. _whatsnew-2.0-convolution:
+
+Improvements to astropy.convolution
+===================================
+
+Convolution has undergone a significant overhaul to make fft and direct
+convolution consistent.  They keyword arguments have changed and the behavior
+of `~astropy.convolution.convolve` is no longer the same as in versions prior to
+2.0 (although `~astropy.convolution.convolve_fft`'s behavior remains unchanged).
+The details are given on the :ref:`astropy convolution <astropy_convolve>`.
+
+.. _whatsnew-2.0-cosmo-def:
+
+No relativistic species by default in cosmological models
+=========================================================
+
+For all of the built in cosmological model types (e.g.,
+:class:`~astropy.cosmology.FlatLambdaCDM`) the default CMB temperature at z=0 is
+now 0K, which corresponds to no contributions from photons or neutrinos (massive
+or otherwise).  This does not affect built in literature models (such as the
+WMAP or Planck models).  The justification is to avoid including mass-energy
+components that the user has not explicitly requested.  This is a non-backwards
+compatible change, although the effects are small for most use cases.
+
+.. _whatsnew-2.0-renamed-removed:
+
+Renamed/removed functionality
+=============================
+
+Several sub-packages have been moved or removed, and these are described in the
+following sections.
+
+astropy.tests.helper.pytest
+---------------------------
+
+The bundled version of ``pytest`` has now been removed, but the
+``astropy.tests.helper.pytest`` import will continue to work properly.
+Affiliated packages should nevertheless transition to importing ``pytest``
+directly rather than from ``astropy.tests.helper``. This also means that
+``pytest`` is now a formal requirement for testing for both Astropy and for
+affiliated packages.
+
+astropy.vo.conesearch
+---------------------
+
+The cone search module has been moved to `Astroquery
+<http://astroquery.readthedocs.io>`_ (0.3.5 and later) and will be removed from
+Astropy in a future version. The API here will be preserved as the "classic" API
+in Astroquery, however some configuration behavior might change; See the
+Astroquery `documentation
+<http://astroquery.readthedocs.io/en/latest/vo_conesearch/vo_conesearch.html>`_
+for new usage details.
+
+astropy.vo.samp
+---------------
 
-.. _whatsnew-2.0-n-submodels:
+The SAMP (Simple Application Messaging Protocol) module, formerly available in
+``astropy.vo.samp``, has now been moved to ``astropy.samp``, so you should
+update any imports to this module.
 
-New ``n_submodels`` shared method in single and compound models
-===============================================================
+REMOVE? - New ``n_submodels`` shared method in single and compound models
```

Comment:
I am neutral on removing this and the other entry below. @nden should make the final decision as she is the `modeling` maintainer.